src/reformatters/dwd/icon_eu/forecast/template_config.py

```diff
@@ -0,0 +1,593 @@
+            "init_time": 1,
+            "lead_time": 120,
+            "latitude": 657,
+            "longitude": 1377,
+        }
+
+        encoding_float32_default = Encoding(
+            dtype="float32",
+            fill_value=np.nan,
+            chunks=tuple(var_chunks[d] for d in self.dims),
+            shards=tuple(var_shards[d] for d in self.dims),
+            compressors=[BLOSC_4BYTE_ZSTD_LEVEL3_SHUFFLE],
+        )
+
+        default_keep_mantissa_bits = 7
+
+        return [
+            # Some of the `comment` text is taken from the DWD Database Reference PDF:
+            # https://www.dwd.de/DWD/forschung/nwv/fepub/icon_database_main.pdf
+            #
+            # We don't include `alb_rad` (shortwave broadband albedo for
+            # diffuse radiation) in the Zarr because, to quote the DWD
+            # Database Reference: "Values over snow-free land points are based
+            # on a monthly mean MODIS climatology." It's much more data-efficient
+            # to just download those monthly means from DWD.
+            DwdIconEuDataVar(
+                name="downward_diffuse_short_wave_radiation_flux_surface",
+                encoding=encoding_float32_default,
+                attrs=DataVarAttrs(
+                    short_name="aswdifd_s",
+                    long_name="Downward diffusive short wave radiation flux at surface (mean over forecast time)",
+                    units="W m-2",
+                    step_type="avg",
+                    standard_name="Mean surface diffuse short-wave radiation flux",  # From ECMWF.
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="aswdifd_s",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="downward_direct_short_wave_radiation_flux_surface",
+                encoding=encoding_float32_default,
+                attrs=DataVarAttrs(
+                    short_name="aswdir_s",
+                    long_name="Downward direct short wave radiation flux at surface (mean over forecast time)",
+                    units="W m-2",
+                    step_type="avg",
+                    comment=(
+                        "Downward solar direct radiation flux at the surface, averaged over forecast time."
+                        " This quantity is not directly provided by the radiation scheme."
+                        " It is aposteriori diagnosed from the definition of the surface net"
+                        " shortwave radiation flux."
+                    ),
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="aswdir_s",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="convective_available_potential_energy",
```

Comment:
is this entire atmosphere?
```suggestion
                name="convective_available_potential_energy_atmosphere",
```

Reply:
Surprisingly, the ICON-EU GRIBs and the [ICON-EU manual](https://www.dwd.de/DWD/forschung/nwv/fepub/icon_database_main.pdf) claim that GRIB_typeOfLevel is set to surface for the `cape` variable. Although I agree that `entire atmosphere` would make more sense!

Reply:
So, if it's OK, I'd propose that I don't change `convective_available_potential_energy` to `convective_available_potential_energy_atmosphere` (because the ICON-EU manual claims that `convective_available_potential_energy` is a _surface_ variable). Unless I've misunderstood?!

Reply:
lets leave as is for now and come back with a later metadata polish and align to CF Conventions. Same for the other possible naming + unit questions in our metadata.